assets/js/components/wp-dashboard/WPDashboardClicks.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WPDashboardClicks component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as MODULES_SEARCH_CONSOLE } from '../../modules/search-console/datastore/constants';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { extractSearchConsoleDashboardData, isZeroReport } from '../../modules/search-console/util';
+import { trackEvent } from '../../util/tracking';
+import DataBlock from '../data-block';
+import PreviewBlock from '../PreviewBlock';
+import ReportError from '../ReportError';
+import ReportZero from '../ReportZero';
+const { useSelect } = Data;
+
+const WPDashboardClicks = () => {
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+
+	const args = {
+		dateRange,
```

Comment:
Should this use start/end dates instead?

Reply:
@kostyalmm Indeed, we should no longer pass `dateRange` (and the related `compareDateRanges` and `multiDateRange`) arguments as they are legacy. Please rely on the `getDateRangeDates` selector in the same way that https://github.com/google/site-kit-wp/blob/develop/assets/js/components/adminbar/AdminBarClicks.js does.